scopus/scopus_reports.py

```diff
@@ -9,17 +9,11 @@ def report(scopus_search, label):
 
     counts = {}  # to count papers per author
     journals = {}  # to count publications per journal
-    author_count = []  # to count number of authors for each paper for a
-                       # histogram
+    author_count = []  # to count a paper's number of authors for a histogram
     paper_cites = {}
     Ncites = 0
     document_types = {}
 
-    # import json
```

Comment:
this block should be removed for sure ;). Do you think it is worthwhile including some code and or the data file containing the impact-factors? I am torn since they change regularly, and not everyone agrees on what they should be used for. Also if I recall, it is pretty tedious to generate that list, and it involved some web scraping.